src/core/index.js

```diff
@@ -39,25 +39,33 @@ import validateNetworkResponseIsWellFormed from "./edgeNetwork/validateNetworkRe
 import isRetryableHttpStatusCode from "./network/isRetryableHttpStatusCode";
 
 // eslint-disable-next-line no-underscore-dangle
-const instanceNamespaces = window.__alloyNS;
+const instanceNames = window.__alloyNS;
 
 const createNamespacedStorage = injectStorage(window);
 
 const { console } = window;
 
+// set this up as a function so that monitors can be added at anytime
+// eslint-disable-next-line no-underscore-dangle
+const getMonitors = () => window.__alloyMonitors || [];
```

Comment:
Let's add a comment OR an entry in the Tech WIKI regarding the interface of the monitors.